msteams-platform/sbs-gs-spfx.yml

```diff
@@ -166,40 +165,31 @@ items:
 
     1. Deselect **Tab**, select **Tab(SPFx)**, and select **OK**.
 
-        :::image type="content" source="../msteams-platform/assets/images/teams-toolkit-v2/create-project-capabilities.png" alt-text="Screenshot showing how to add capabilities to your new app.":::
+        :::image type="content" source="~/assets/images/teams-toolkit-v2/spfxTab-spfx.png" alt-text="bot home" border="true":::
```

Comment:
a heads up that there's a small string change on this screenshot, which I will provide the new screenshot tomorrow